src/main/java/seedu/manager/Main.java

```diff
@@ -41,4 +48,28 @@ private static void runCommandLoop() {
             }
         }
     }
+
+    /**
+     * Loads events from file and handles exceptions.
+     */
+    private static void loadData() {
+        try {
+            storage.loadEvents(events); // Load events using storage instance
+            ui.showMessage("Events loaded successfully.");
+        } catch (IOException e) {
+            ui.showErrorMessageToUser(e); // Use showErrorMessageToUser
+        }
```

Comment:
I think these comments are redundant and can be removed, as it is obvious enough to me what the code is doing?